.travis.yml

```diff
@@ -29,8 +29,11 @@ notifications:
       on_success: change
       on_failure: always
 
-cache: 
-    - apt
+cache:
+  apt: true
+  ccache: true
+  directories:
+  - "$HOME/.ccache"
```

Comment:
https://docs.travis-ci.com/user/caching/#ccache-cache implies that the last line should not be necessary.